pr-data.csv

```diff
@@ -1391,14 +1392,15 @@ https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-s
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-streaming-java,org.apache.flink.streaming.runtime.operators.windowing.MergingWindowSetTest.testMergeLargeWindowCoveringMultipleWindows,ID,Rejected,https://github.com/apache/flink/pull/11266,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-connectors/flink-connector-kafka-base,org.apache.flink.table.descriptors.KafkaTest.testValidation,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/8
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-common,org.apache.flink.table.expressions.ExpressionTest.testValueLiteralString,ID,Accepted,https://github.com/apache/flink/pull/11269,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapField,ID,,,
+https://github.com/apache/flink,1845aa8e6fc094ed69a145b50ac295e058d8c695,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapField,ID,,,
+https://github.com/apache/flink,1845aa8e6fc094ed69a145b50ac295e058d8c695,flink-table/flink-table-planner,org.apache.flink.table.planner.expressions.ScalarFunctionsTest.testPi,ID,,,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapLiteral,ID,,,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapTypeCasting,ID,,,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.SqlExpressionTest.testValueConstructorFunctions,ID,,,
+https://github.com/apache/flink,1845aa8e6fc094ed69a145b50ac295e058d8c695,flink-table/flink-table-planner,org.apache.flink.table.expressions.SqlExpressionTest.testValueConstructorFunctions,ID,,,
```

Comment:
Again, unclear why just changing the commit SHA for this test. The best would be to fix this test and others.

Reply:
These are merge changes, nothing I did.